backend/Dockerfile

```diff
@@ -10,9 +10,5 @@ RUN ./gradlew shadowJar
 FROM openjdk:14-alpine
 RUN apk update
 RUN apk add python3
-RUN ln -s /usr/bin/python3 /usr/bin/python
-RUN wget https://yt-dl.org/downloads/latest/youtube-dl -O /usr/local/bin/youtube-dl
-RUN chmod a+rx /usr/local/bin/youtube-dl
-COPY --from=build /app/build/libs/yt-sync-all.jar ./yt-sync-all.jar
```

Comment:
` COPY --from=build /app/build/libs/yt-sync-all.jar ./yt-sync-all.jar` needs to be inside the Dockerfile